android/KMEA/app/src/main/java/com/tavultesoft/kmea/cloud/impl/CloudCatalogDownloadCallback.java

```diff
@@ -146,7 +146,8 @@ public void processCloudReturns(Dataset aDataSet, CloudCatalogDownloadReturns js
       return;
     }
 
-    List<LexicalModel> lexicalModelsArrayList = CloudDataJsonUtil.processLexicalModelJSON(jsonTuple.lexicalModelJSON);
+    boolean fromKMP = false;
```

Comment:
```suggestion
    final boolean fromKMP = true;
```

If this never changes, make it clear that it's a constant :)